discord/commands/commands.py

```diff
@@ -700,17 +700,31 @@ def _update_copy(self, kwargs: Dict[str, Any]):
     'TextChannel': ChannelType.text,
     'VoiceChannel': ChannelType.voice,
     'StageChannel': ChannelType.stage_voice,
-    'CategoryChannel': ChannelType.category
+    'CategoryChannel': ChannelType.category,
+    'Thread': ChannelType.public_thread
 }
 
+class ThreadOption:
+    def __init__(self, thread_type: Union[Literal["public"], Literal["private"], Literal["news"]]):
+        type_map = {
+            "public": ChannelType.public_thread,
+            "private": ChannelType.private_thread,
```

Comment:
News thread?

Reply:
apparently it exists

<img width="822" alt="Screen Shot 2022-01-05 at 4 08 17 PM" src="https://user-images.githubusercontent.com/73825066/148182489-0578fcea-f904-44c1-82f5-33d1c91a3b62.png">

Reply:
public becomes automatically a news thread when it is in an announcement channel